docs/sdk/src/guides/enable_elastic_scaling_mvp.rs

```diff
@@ -135,10 +135,14 @@
 //!      ```
 //!     Note: for a parachain which measures time in terms of its own block number, changing block
 //!     time may cause complications, requiring additional changes.  See here more information:
-//!     [`crate::guides::async_backing_guide#timing-by-block-number`].
+//!     [`async_backing_guide#timing-by-block-number`](crate::guides::async_backing_guide#timing-by-block-number).
```

Comment:
```suggestion
//!     [`async_backing_guide#timing-by-block-number`].
```

I think for these final display should be like this, not containing the header.

Reply:
Thanks for your suggestion.
I have incorporated those changes. 
Let me know if thats fine